<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="outDeeplinks">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>721</width>
      <height>251</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="apkSelect">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
        <string>Select APK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>16</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>16</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>2.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extract">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Possible deeplinks</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="processResult">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>721</width>
      <height>160</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectedFile">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>511</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
      <widget class="QCheckBox" name="asIntent">
          <property name="geometry">
              <rect>
                  <x>230</x>
                  <y>60</y>
                  <width>161</width>
                  <height>24</height>
              </rect>
          </property>
          <property name="text">
              <string>as &quot;intent&quot;</string>
          </property>
      </widget>
      <widget class="QCheckBox" name="clearCache">
          <property name="geometry">
              <rect>
                  <x>230</x>
                  <y>90</y>
                  <width>161</width>
                  <height>24</height>
              </rect>
          </property>
          <property name="text">
              <string>overwrite cache</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
